specs/engage-digital_openapi3.yaml

```
@@ -97,6 +98,143 @@ tags:
+        
+        
+        Authorization: only users that are techteam.
+      operationId: activateTopology
+      parameters:
+        - in: path
+          name: topologyId
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Topology'
+          description: Success
+      summary: Activating a topology
+      tags:
+        - Topologies
+  '/topologies/{topologyId}':
+    put:
+      description: >-
+        This method updates an existing topology from given attributes and renders it in case of success.
+        
+        
+        Authorization: only users that have the right to manage topologies. Topology must be inactive or the response will return an error.
+      operationId: updateTopology
+      parameters:
+        - in: path
+          name: topologyId
+          required: true
+          schema:
+            type: string
+      requestBody:
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              $ref: '#/components/schemas/CreateUpdateTopology'
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Topology'
+          description: Success
+      summary: Updating a topology
+      tags:
+        - Topologies
+    delete:
+      description: >-
+        This method destroys an existing topology. It renders topology itself. It renders a 404 if id is invalid.
+        
+        
+        Authorization: only users that are techteam.
+      operationId: deleteTopology
+      parameters:
+        - in: path
+          name: topologyId
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Topology'
+          description: Success
+      summary: Deleting a topology
+      tags:
+        - Topologies
+    get:
+      description: >-
+        This method renders a topology from given id.
+      
+      
+        Authorization: only users that have the right to manage topologies.
+      operationId: getTopology
+      parameters:
+        - in: path
+          name: topologyId
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Topology'
+          description: Success
+      summary: Getting a topology from its id
+      tags:
+        - Topologies
+  /topologies:
+    post:
+      description: >-
+        This method creates a topology. In case of success it renders the topology, otherwise, it renders an error (422 HTTP code).
+      
+          Authorization: only users that are superadmin.

```

Indentation and permission change

```suggestion
        Authorization: only users that have the right to manage topologies.
```